apps/www/registry/default/ui/skeleton.tsx

```diff
@@ -1,15 +1,15 @@
+import * as React from "react"
```

Comment:
A couple of components were missing this import (I can't recall offhand what the other one was). I've added the import for the sake of consistency (and also to eliminate the squiggly lines I get in WebStorm for using a UMD global variable 🤪)